pom.xml

```diff
@@ -63,6 +57,67 @@
 
 	<profiles>
 		<profile>
+			<id>ossrh</id>
+			<distributionManagement>
```

Comment:
@ansell are you sure that's necessary? The way I read the Sonatype guide you only need this if you do not use the nexus staging plugin.